oqsprov/oqsprov.c

```diff
@@ -289,18 +289,22 @@ static const OSSL_PARAM *oqsprovider_gettable_params(void *provctx)
     return oqsprovider_param_types;
 }
 
+#define OQS_PROVIDER_BUILD_INFO_STR "OQS Provider v." OQS_PROVIDER_VERSION_STR " based on liboqs v." OQS_VERSION_TEXT
+
 static int oqsprovider_get_params(void *provctx, OSSL_PARAM params[])
 {
     OSSL_PARAM *p;
+    char buildinfo[512];
 
+    snprintf(buildinfo, 512, "OQS Provider v.%s based on liboqs v.%s", OQS_PROVIDER_VERSION_STR, OQS_VERSION_TEXT);
```

Comment:
Can be removed now.

Reply:
:-( Nothing too small to not be forgotten. Thanks.